### PR TITLE
CC clothing armor buff + description and name changes

### PR DESCRIPTION
**Pull request by E-MonaRhg**
## What Does This PR Do

Gives the SolGov Lieutenant, SOO, Navy Officer, Navy Captain (+ other CC roles that use their uniforms) and Blueshield security officer's jumpsuit grade armor + job specific Formal Uniform names: "Formal Nanotrasen Representative's Uniform", "Formal Blueshield's Uniform" and "Formal Magistrate's Uniform" instead of just "Formal Nanotrasen Uniform".

## Why It's Good For The Game

CC uniforms had "It's got exotic materials for protection." in the description yet no armor, now have armor to be less confusing. And I consider that the fact that Formal Blueshield's Uniform didn't have any armor compared to the normal blueshield's uniform was somewhat of a noob-trap for the Blueshields that wanted to look fancy. It also has "It's got exotic materials for protection." in the description now.

## Changelog
:cl:
tweak: Gave armor to several CC uniforms
tweak: Changed the names and descriptions of karma CC role formal uniforms + the NTR's hat
/:cl:

### CODE REVIEW

**Review by variableundefined — APPROVED** [review]
I'm fine with this if it just brings the formal uniform in line with the normal one for protection

File: code/modules/clothing/under/miscellaneous.dm [hunk 1]
```diff
@@ -74,6 +74,7 @@
 	desc = "A comfortable and durable combat uniform worn by Sol Federation Marine Forces. This one has additional insignia on its shoulders."
 	icon_state = "solgovc"
 	item_color = "solgovc"
+	armor = list("melee" = 10, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 30, "acid" = 30)
```
Comment (farie82) [inline code/modules/clothing/under/miscellaneous.dm:77]: Spacing issue here and the others:
```suggestion
	armor = list("melee" = 10, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 30, "acid" = 30)
```

File: code/modules/clothing/under/miscellaneous.dm [hunk 2]
```diff
@@ -105,6 +106,7 @@
 	icon_state = "navy_gold"
 	item_state = "navy_gold"
 	item_color = "navy_gold"
+	armor = list("melee" = 10, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 30, "acid" = 30)
```
Comment (farie82) [inline code/modules/clothing/under/miscellaneous.dm:109]:
```suggestion
	armor = list("melee" = 10, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 30, "acid" = 30)
```

File: code/modules/clothing/under/miscellaneous.dm [hunk 3]
```diff
@@ -113,19 +115,21 @@
...
```
Comment (farie82) [inline code/modules/clothing/under/miscellaneous.dm:118]:
```suggestion
	armor = list("melee" = 10, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 30, "acid" = 30)
```

File: code/modules/clothing/under/miscellaneous.dm [hunk 4]
```diff
@@ -113,19 +115,21 @@
...
 
 /obj/item/clothing/under/rank/centcom/blueshield
-	desc = "Gold trim on space-black cloth, this uniform bears \"Close Protection\" on the left shoulder."
-	name = "\improper Formal Nanotrasen Uniform"
+	desc = "Gold trim on space-black cloth, this uniform bears \"Close Protection\" on the left shoulder. It's got exotic materials for protection."
+	name = "\improper Formal Blueshield's Uniform"
 	icon_state = "officer"
 	item_state = "g_suit"
 	item_color = "officer"
+	armor = list("melee" = 10, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 30, "acid" = 30)
```
Comment (farie82) [inline code/modules/clothing/under/miscellaneous.dm:127]:
```suggestion
	armor = list("melee" = 10, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 30, "acid" = 30)
```

**Review by farie82 — CHANGES_REQUESTED** [review]
Small nitpick. Looks good otherwise

**Comment by E-MonaRhg** [comment]
Spacing issues near "energy" were due to me copying the armor line from the normal blueshield uniform

**Review by farie82 — APPROVED** [review]

**Review by Kyep — APPROVED** [review]

**Pull request merged**